src/shadows-and-depth.md

```diff
@@ -0,0 +1,106 @@
+# Shadows and depth
```

Comment:
Kiarokh wrote:
> Do we have to say anything about how they can import these css variables? Or is it a given that they know?

Not in this guide at least. For those building add-ons, they won't have to import these at all, since that's done by the client. We might want to document how to do _that_ somewhere, but I don't think that belongs in this design guide.